packages/api/public/spec/schemas/parameters.yml

```
@@ -0,0 +1,24 @@
+languageCode:
+  name: code
+  in: path
+  description: IETF code of the language
+  required: true
+  schema:
+    type: string
+  example: es
+verseId:
+  name: verseId
+  in: path
+  description: The verse number within the chapter. In the format of `BBCCCVVV`.
```

```suggestion
  description: The verse number within the chapter. In the format of `BBCCCVVV` (**B**ook, **C**hapter, **V**erse).
```
Just to be very explicit.

---

(I did test this in my local environment, swagger renders the `**Bold**` Markdown fine)

---

Good idea. That's a concise way to clarify